inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/embedding_bag_offsets_sum.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/embedding_bag_offsets_sum.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+//        InferenceEngine::Precision::U8,
+//        InferenceEngine::Precision::I8
```

Comment:
I think we should test other precisions as well

Reply:
Fixed